js/views/settingsVw.js

```diff
@@ -745,11 +747,14 @@ module.exports = pageVw.extend({
     var self = this,
         form = this.$("#generalForm"),
         cCode = this.$('#currency_code').val(),
+        bitcoinUnit = this.$("#generalForm input[name=bitcoinUnit]:checked").val(),
         $saveBtn = this.$('.js-saveGeneral');
 
     $saveBtn.addClass('loading');
 
     localStorage.setItem('NSFWFilter', this.$("#generalForm input[name=nsfw]:checked").val());
+    localStorage.setItem('BitcoinUnit', bitcoinUnit);
```

Comment:
Please see my comment above about centralizing this in App.js, after which you could remove these two lines and just call `app.setBitcoinUnit`.